adapters/smaato/smaato.go

```diff
@@ -57,7 +57,8 @@ type bidRequestExt struct {
 
 // bidExt defines Bid.Ext object for Smaato
 type bidExt struct {
-	Duration int `json:"duration"`
+	Duration int      `json:"duration"`
+	Curls    []string `json:"curls"`
```

Comment:
What is the main purpose of adding the Curls field to the bidExt structure? Can you provide examples of situations where it will be used?

Reply:
is it click url?

Reply:
@przemkaczmarek 
yes it is click urls array
use case is to track ads clicks - both our own and potentially demand partners - and since there is no ortb specific field for this we decided to use `ext`